types/braintree-web/paypal-checkout.d.ts

```diff
@@ -43,15 +43,11 @@ export interface PayPalCheckoutLoadPayPalSDKOptions {
      * wish to use a different intent when calling createPayment, make sure it matches here. If sale
      * is used, it will be converted to capture for the PayPal SDK. If the vault: true param is used,
      * no default intent will be passed.
-     *
-     * @default 'authorize'
```

Comment:
Had to remove for tests to pass, deprecated

Reply:
Why did you have to remove it? Other people didn't have this problem and it should definitely stay in the JS doc.

Reply:
Left a reply below 🙏